Removed more copyright materials (bar signs)

Peptide90 opened the pull request:
Removed wretchedhive bar sign and altered LV426 to a spider.

If anyone can spot or think of anything else copyright related let me know and I can bundle it in with this PR.

![lv426](https://user-images.githubusercontent.com/78795277/147852839-5739a639-de46-4f26-a738-c2a7dbf38f7f.png)

Also updated names on some posters to remove xeno reference and corrected the meta.json to correct license on bar signs.

PaulRitter reviewed (changes requested):
you gotta adjust the barsign.rsi/meta.json to reflect the deleted state

Peptide90 commented:
> you gotta adjust the barsign.rsi/meta.json to reflect the deleted state

My bad, forgot about that. Fixed now and corrected some other issues and copyright in posters.

Phantom-Lily commented:
"Richard" item probably falls under this.
